File: SecondYear/ApoorvaSri123_ApoorvaSrivastava_2226it1116_2/Week_3/minimum_and_maximum.c
#include <stdio.h>

int findMin(int array[], int N) {
    int min = array[0];
    for (int i = 1; i < N; i++) {
        if (array[i] < min) {
            min = array[i];
        }
    }
    return min;
}

int findMax(int array[], int N) {
    int max = array[0];
    for (int i = 1; i < N; i++) {
        if (array[i] > max) {
            max = array[i];
        }
    }
    return max;
}

int main() {
    int N;
    printf("Enter the number of elements: ");
    scanf("%d", &N);

    int array[N];

    printf("Enter the elements of the array:\n");
    for (int i = 0; i < N; i++) {
        scanf("%d", &array[i]);
    }

    int min = findMin(array, N);
    int max = findMax(array, N);

    printf("Minimum: %d\nMaximum: %d\n", min, max);

    return 0;
}

File: SecondYear/Divyansh670_DivyanshSrivastav_2226it1029_2/Week_3/minimum_and_maximum.java
import java.util.Scanner;

public class minimum_and_maximum {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        int N = scanner.nextInt();

        int[] array = new int[N];

        for (int i = 0; i < N; i++) {
            array[i] = scanner.nextInt();
        }

        int min = findMin(array);
        int max = findMax(array);

        System.out.println( min +" "+ max);
        

        scanner.close();
    }

    public static int findMin(int[] array) {
        int min = array[0];
        for (int i = 1; i < array.length; i++) {
            if (array[i] < min) {
                min = array[i];
            }
        }
        return min;
    }

    public static int findMax(int[] array) {
        int max = array[0];
        for (int i = 1; i < array.length; i++) {
            if (array[i] > max) {
                max = array[i];
            }
        }
        return max;
    }
}
